docs/compose.md

```diff
@@ -39,8 +39,32 @@ releases will increase the amount of customization that is available.
 It is also possible to [build Triton](build.md#building-triton)
 yourself to get more exact customization.
 
-The two Docker images used for customization are retrieved using the
-following commands.
+## Use the compose.py script
+
+The `compose.py` script uses the existing NGC containers 
+```
+nvcr.io/nvidia/tritonserver:<xx.yy>-py3-min
+nvcr.io/nvidia/tritonserver:<xx.yy>-py3
+```
+to build the new custom container. The resulting container version will be `xx.yy` as it inherits from the pre-mentioned builds.
+
+### Customizing backends and repo-agents
```

Comment:
I would not create a new section here but just continue the previous paragraph...
"`compose.py` provides `--backend` and `--repoagent` options that allow you to specify which backends and repository agents you want to include in the custom image. The --enable-gpu flag indicates that you want to create an image that supports NVIDIA GPUs. For example, the following creates a new docker image that contains only the TensorFlow 1 and TensorFlow 2 backends and the checksum repository agent.